pipeline/masked/psdbinner.py

```diff
@@ -2,9 +2,9 @@
 
 from ..psdbinner import PSDBinner
 from typing import List, Tuple
+from sklearn.base import BaseEstimator, TransformerMixin  # type: ignore
 
-
-class MaskedPSDBinner(PSDBinner):
+class MaskedPSDBinner(PSDBinner,BaseEstimator):
```

Comment:
@ahmad11111111111 

Thanks for making this pull request. Is there a reason that you're reassigning the `BaseEstimator` class here? It is already assigned to the `PSDBinner`, which means by inheriting from `PSDBinner` the `MaskedPSDBinner` will also inherit from this class